firewood/src/manager.rs

```diff
@@ -65,7 +69,7 @@ type CommittedRevision = Arc<NodeStore<Committed, FileBacked>>;
 type ProposedRevision = Arc<NodeStore<Arc<ImmutableProposal>, FileBacked>>;
 
 #[derive(Debug)]
-pub(crate) struct RevisionManager {
+pub(crate) struct RevisionManager<T: RootStore> {
```

Comment:
```suggestion
pub(crate) struct RevisionManager<T> {
```
(Check all remaining structs, chances are they don't need bounds on T; I'm not mentioning any remaining ones)